Adapt documentation to the new side menu. (#6690)
wazuh/wazuh-documentation: source/user-manual/wazuh-dashboard/custom-branding.rst

Custom logos in the Wazuh dashboard 
-----------------------------------

> To use your own logos in the Wazuh dashboard, click on the `App main logo` and go to **App Settings**. Under the **Custom branding** section, set up the following properties:

Reviewer: ```suggestion
To use your own logos in the Wazuh dashboard, click on **Indexer/dashboard management** > **App Settings**. Under the **Custom branding** section, set up the following properties:
```

Custom branding of the PDF reports
----------------------------------

> To customize the PDF reports, click on the `App main logo` and go to **App Settings**. Under the **Custom branding** section, set up the following properties:

Reviewer: ```suggestion
To customize the PDF reports, click **Indexer/dashboard management** > **App Settings**. Under the **Custom branding** section, set up the following properties:
```
